errors.go

```diff
@@ -31,6 +31,7 @@ type Error struct {
 	Message     string            `json:"message"`
 	Params      map[string]string `json:"params"`
 	StackFrames stack.Stack       `json:"stack"`
+	err         error
```

Comment:
This is unexported so that it isn't included when the terror is serialised. Support for unwrapping errors only works within a service and not across service boundaries.

Reply:
Can you add that comment to the code, and include the 'why' as well? 🙏